### ISSUE TITLE
list with variable types, default values not working

### ISSUE THREAD

**Issue by speexy** [netlify/netlify-cms#2116]
**Is your feature request related to a problem? Please describe.**
Feature. Right now, when I set default values within a list of variable type, it will be ignored.
example:
```
  - name: "posts"
    label: "Posts ✒"
    fields:
      - {label: "Content", name: "content", widget: "list", types: [
          {label: "Global Goals", name: "globalgoals", widget: "object", fields: [
            {label: "Global Goals Image", name: "gsdgimage", widget: "image", default: "/media/gsdg.jpg"},
            {label: "Global Goals Text", name:"gsdgtext", widget: "markdown", default: "17 awesome goals that everybody loves"}
          ]}
        ]}
```
--> when I create a new post and add the block "Global Goals" there are no default values

**Describe the solution you'd like**
I'd like to be able to have list elements with default values, as the user has content blocks that need to be added with minimal changes to the content. 

**Additional context**
This is somewhat related to #981 
I personally think components with default values are much more elegant than copying old content though (which is what our users do a LOT in wordpress)

**Comment by tomrutgers** [comment]
@speexy Duplicate of #2030?

**Comment by speexy** [comment]
@tomrutgers, no, not really. I am fine with nothing in particular happening when the user does NOT use the widget. As is said there, that is related to gatsby/graphql and the workaround posted there is a good idea for now.  (It would still be awesome to have a solution for that too though!)  

This is about when a user DOES use the widget and wants it to be pre-filled with default values. 

In #2030 I would want the defaults to be empty strings or null or something like that so nothing gets rendered. Here in #2116 I would want actual content that does get rendered.

Combined this would come down to: The unused widget should default to empty/null and the used widget should default to pre-set content.

**Comment by tomrutgers** [comment]
Gotcha, just wanted to make sure! 👍

**Comment by barthc** [comment]
This issue already fixed in #2395
